views/js/controller/creator/views/rubricblock.js

```diff
@@ -82,10 +77,41 @@ define([
             }
 
             /**
-             * Update the HTML editor
+             * Ensures an html content is wrapped by a container tag.
+             * @param {String} html
+             * @returns {String}
              */
-            function updateContent() {
-                var html = Dom2QtiEncoder.encode(rubricModel.content);
+            function ensureWrap(html) {
+                html = (html || '').trim();
+                if (html.charAt(0) !== '<' && html.substr(3) !== '<br') {
```

Comment:
Why only `<br>` you can have the situation with any inline tag, no ? So you can have 
`<a href="#">foo</a> some other text`